hack/DebugWindowsNode.ps1

```diff
@@ -57,13 +57,12 @@ Describe "Docker is installed" {
     }
 
     It "Should be a supported version" {
-        $supportedVersions = @(
-            "19.03*",
-            "19.04*",
-            "19.1*",
-            "20*"
-        )
-        $supportedVersions -contains $global:DockerVersion.Server.Version | Should Be $true
+        $supported = $false
+        if ($global:DockerVersion.Server.Version -match "\d+\.\d+\.\d+") {
```

Comment:
is this additional matching required?  I think this whole if statement could be simplified to

```
$supported = [version]$global:DockerVersion.Server.Version -ge [version]"19.3.0"
```

if the version isn't semantic it will error

Reply:
Yes, it does.
```
PS C:\> [version]"20.10.0-rc1"
InvalidArgument: Cannot convert value "20.10.0-rc1" to type "System.Version". Error: "Input string was not in a correct format."
```

Reply:
interesting, looks like `[version]` doesn't handle the full sem ver spec